### PR TITLE
Add support for Raw File ingestion

### PR DESCRIPTION
**Pull request by apoorva918**
This PR adds a new service to Pravega Sensor Collector for Raw file ingestion. 
The **RawFileIngestService** works similar to LogFileIngestService. No data parsing is done, files are written to stream as byte arrays. 

- Each file is sent as a single event
- Added directory traversal: Files can be read and deleted after ingestion from within subfolders

**Windows Service support:**
Windows Service Wrapper (winsw) is used to run the java application as a windows service.
Required files:

- PravegaSensorCollectorApp.xml
- WinSW.exe or WinSW.zip from the distribution, and renamed the .exe to PravegaSensorCollectorApp.exe

CLI commands to run the Pravega Sensor Collector as a windows service:

PravegaSensorCollectorApp.exe install 
PravegaSensorCollectorApp.exe start 
PravegaSensorCollectorApp.exe restart 
PravegaSensorCollectorApp.exe status 
PravegaSensorCollectorApp.exe stop 
PravegaSensorCollectorApp.exe uninstall

**Testing:**
Tested with 9 parquet files. All files were read and ingested.

### CODE REVIEW

File: pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/rawfile/EventGenerator.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+     * @param inputStream
+     * @param firstSequenceNumber
+     * @return next sequence number, end offset
+     */
+    protected Pair<Long, Long> generateEventsFromInputStream(CountingInputStream inputStream, long firstSequenceNumber, Consumer<RawFileWriterEvent> consumer) throws IOException {
+        
+        long nextSequenceNumber = firstSequenceNumber;
+        try{    
+            byte[] byteArray = inputStream.readAllBytes();
+            //TODO: Batching
```
Comment (sachin-j-joshi) [inline pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/rawfile/EventGenerator.java:68]: what is this TODO?

File: pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/rawfile/FileNameWithOffset.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+/**
+ * File name and file size
+ */
+public class FileNameWithOffset implements Comparable<FileNameWithOffset> {
+    public final String fileName;
+    /**
+     * In some contexts, this is the size of the file.
+     * In the future, this will represent the offset in the file for incrementally ingesting growing log files.
+     * This is partially implemented today.
+     * TODO: Clarify usage of offset.
```
Comment (sachin-j-joshi) [inline pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/rawfile/FileNameWithOffset.java:23]: what is this TODO?

File: pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/rawfile/EventGenerator.java (hunk 1 above)
Comment (apoorva918) [reply pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/rawfile/EventGenerator.java:68]: We are writing each file as one event and so I've removed the SAMPLES_PER_EVENT parameter.
Added this TODO to specify future work for batching files/events.

File: pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/rawfile/FileNameWithOffset.java (hunk 2 above)
Comment (apoorva918) [reply pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/rawfile/FileNameWithOffset.java:23]: This was brought over from https://github.com/apoorva918/pravega-sensor-collector/blob/57f835d505c1d9f74b8e159f6a283fe2aa554c1a/pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/file/FileNameWithOffset.java#L20C8-L20C8
I can remove it if its not required.

**Review by sachin-j-joshi — APPROVED** [review]
LGTM

**Pull request merged**